src/scripts.js

```diff
@@ -102,9 +107,11 @@ const loadPage = () => {
     newUser = new User(randomUser);
     cookbook = new RecipeBox(data[1]);
     recipe = new Recipe(data[1]);
+    pantry = new Pantry(newUser)
     groceryStore = data[0];
     displayHomePage();
   })
+  console.log('here', recipe)
```

Comment:
We gotta remember to get rid of this console.log()! Awesome way to check our stuff, though. Genius!